templates/bundles/PrestaShopBundle/Admin/Sell/Catalog/Categories/Blocks/form.html.twig

```diff
@@ -26,7 +26,7 @@
 {% import '@PrestaShop/Admin/macros.html.twig' as ps %}
 
 {% block category_form_block %}
-{{ form_start(categoryForm) }}
+{{ form_start(categoryForm, {'attr': {'data-id': categoryId|default(0)}}) }}
```

Comment:
```suggestion
{{ form_start(categoryForm, {'attr': {'data-object-id': categoryId|default(0)}}) }}
```

it's not a form id, the id is related to the object. We have to specify it in the name